cmd/maya-apiserver/cstor-operator/cspc/handler.go

```diff
@@ -160,3 +170,23 @@ func (pc *PoolConfig) create(pendingPoolCount int, cspc *apis.CStorPoolCluster)
 	}
 	return nil
 }
+
+func (pc *PoolConfig) createDeployForCSPList(cspList []apis.NewTestCStorPool) {
+	for _, cspObj := range cspList {
+		cspObj := cspObj
+		pc.createDeployForCSP(&cspObj)
+	}
+}
+
+func (pc *PoolConfig) createDeployForCSP(csp *apis.NewTestCStorPool) {
+	deployObj, err := pc.GetPoolDeploySpec(csp)
+	if err != nil {
+		glog.Errorf("could not get deployment spec for csp {%s}:{%s}", csp.Name, err.Error())
```

Comment:
lets record the error on CSPC CR as an event.. this can be done in next PR along with other error paths